packages/webpack/lib/plugins/render.js

```diff
@@ -20,6 +20,9 @@ exports.RenderPlugin = class RenderPlugin {
         )
       );
       compiler.hooks.compilation.tap('RenderPlugin', (compilation) => {
+        if (compilation.compiler.isChild()) {
```

Comment:
I did not know `Compiler` instances had an `isChild()` method - nice find!

Could we not also use it in the [`StatsPlugin`](https://github.com/untool/untool/blob/master/packages/webpack/lib/plugins/stats.js#L55)?

//cc @ZauberNerd